CHANGELOG.md

```diff
@@ -7,3 +7,17 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ## Added
 - Basic Skel
 - missing PR template
+
+## [0.0.2] - 2016-08-11
```

Comment:
For the time being I have dropped the versioning as requested. I can easily restore it if desired just let me know. With the latest changes and depending on how the rubygems thing goes perhaps the first version released here should be newer than 0.0.2.